pkg/validator/pod.go

```diff
@@ -54,7 +55,7 @@ func ValidatePod(podConf conf.Configuration, pod *corev1.PodSpec) PodResult {
 
 func (pv *PodValidation) validateContainers(containers []corev1.Container, pRes *PodResult, podConf *conf.Configuration, isInit bool) {
 	for _, container := range containers {
-		cRes := ValidateContainer(podConf, &container, isInit)
+		cRes := ValidateContainer(podConf, &container, isInit, pRes)
```

Comment:
While you're in here, probably makes sense to match the parameter order with the above, i.e. `(&container, pres, podConf, isInit)`